src/v/transform/api.cc

```diff
@@ -827,7 +827,7 @@ model::cluster_transform_report service::compute_default_report() {
               transform,
               {
                 .id = model::partition_id(i),
-                .status = state::unknown,
+                .status = transform.paused ? state::inactive : state::unknown,
```

Comment:
Can you add a lengthy comment here explaining what we had talked about **why** we do this in our pairing session?

I think it's important to note the ramifications that the behavior here is dependent and tied to how we model state in the `transform::manager`

Reply:
yeah absolutely

Reply:
done